src/api/app/controllers/webui/request_controller.rb

```diff
@@ -209,10 +209,6 @@ def accept_request
 
   def forward_request_to(fwd)
     tgt_prj, tgt_pkg = params[fwd].split('_#_') # split off 'forward_' and split into project and package
-    description = @req.description
-    description ||= ""
-    who = @req.creator
-    description += ' (forwarded request %d from %s)' % [params[:number], who]
```

Comment:
I don't know, if it should be used it is already wrong because it is overwritten :confused: